packages/commerce-sdk-react/README.md

```diff
@@ -1,24 +1,51 @@
 # Commerce SDK React
 
-A library of React hooks for fetching data from Salesforce B2C Commerce.
+<p align="center">
+A collection of React hooks for <b>fetching</b>, <b>caching</b>, and <b>mutating data</b> from the <b><a href="https://developer.salesforce.com/docs/commerce/commerce-api/overview">Salesforce B2C Commerce API</a></b> (SCAPI).
+The library contains declarative, always-up-to-date auto-managed <b>queries</b> and <b>mutations</b> for a more convenient developer experience.
+</p>
 
-## Documentation
+## 🎯 Features
 
-The full documentation for PWA Kit and Managed Runtime is hosted on the [Salesforce Developers](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/overview) portal.
+- [react-query](https://tanstack.com/query/latest/docs/react/overview) and [commerce-sdk-isomorphic](https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic) integration
+- Shopper authentication & token management via [SLAS](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-login)
+- SSR data fetching React Hooks
+- Phased Launch support ([plugin_slas](https://github.com/SalesforceCommerceCloud/plugin_slas) compatible)
+- [TypeScript](https://www.typescriptlang.org/) Support
+- Multi-site and locale/currency support
+- Request deduplication
+- Built-in caching for easy state management
+  - automatic cache invalidations/updates via the library's built-in mutations
+  - automatic cache key generation
 
+## ⚙️ Installation
 
-## PWA-Kit Integration
+```bash
+npm install @salesforce/commerce-sdk-react @tanstack/react-query
+```
 
-> To integration this library with your PWA-Kit application you can use the `CommerceApiProvider` directly given that you use the `withReactQuery` higher order component to wrap your `AppConfig` component. Below is a snippet of how this is accomplished.
+## ⚡️ Quickstart (PWA Kit)
 
-```
+To integrate this library with your PWA Kit application you can use the `CommerceApiProvider` directly assuming that you use the `withReactQuery` higher order component to wrap your `AppConfig` component. Below is a snippet of how this is accomplished.
```

Comment:
added pwa kit version.

I do think it's worth mentioning `withReactQuery` since the package is inside the monorepo and the primary use case is `withReactQuery`. This highlights the integration and gives more exposure of the SDK integration, otherwise developers might not know `withReactQuery` and things could be a bit black boxy for them. That's why i left the `withReactQuery` examples in the doc.

Reply:
I'll meet you half way on this one 👍